index.html

```diff
@@ -20,10 +20,70 @@
     <link rel="stylesheet" href="style.css" />
   </head>
   <body>
-    <!--
+    <header>
+      <ul class="header">
+        <h1 class="logo">Fitzone</h1>
+        <h2 class="nav1">HOME</h2>
+        <h2 class="nav2">ABOUT US</h2>
+        <h2 class="nav3">MEMBERSHIP</h2>
+        <h2 class="nav4">LOCATIONS</h2>
+        <h2 class="signup">SIGN UP</h2>
+        <h2 class="login">LOGIN</h2>
+      </ul>
+    </header>
+    <main>
+      <div class="banner">
+        <p class="p1">
+          Fitzone is commited to providing the best possible<br> 
+          experience that a gym can offer. We provide our<br>
+          members with access to high quality equipment, as well<br>
+          as a welcoming and judgement free environment designed<br>
+          to allow you to improve yourself at a pace that's right for you.<br>
+          Browse our membership options and begin your journey to be <br>
+          the best you.
+        </p>
+        <button type="submit" class="b1">BECOME A MEMBER</button>
+        <form class="info">
+          <label for="name">Name:</label>
+          <input type="text" id="fullname" name="fullname" autocomplete="name" />
+          <label for="email">Email:</label>
+          <input type="text" id="email" name="email" autocomplete="email" />
+        </form>
+      </div>
+      <div class="sec1">
+        <h2 class="title1">TRAINING PROGRAMS</h2>
+        <p class="p2">
+          Our expert trainers are here to provide<br> you with specialized programs suited<br> to your needs.
+        </p>
+        <p class="b2">BROWSE PROGRAMS</p>
+        <img class="i1" src="Trainer.png" alt="Trainer">
+      </div>
 
-      YOU HAVE TO MODIFY THE HTML TO BUILD YOUR AWESOME PAGE
+      <div class="sec2">
+        <h2 class="title2">LOCATIONS</h2>
+        <p class="p3">
+          Our state-of-the-art facilities have<br>everything you need to push yourself<br>to the next level.
+        </p>
+        <p class="b3">SEE LOCATIONS</p>
+        <img class="i2" src="gym.png" alt="gym interrior">
+      </div>
 
-    -->
+      <div class="sec3">
+        <h2 class="title3">JOIN THE TEAM</h2>
+        <p class="p4">
+          Join the team and help our members<br>become their best selves.
+        </p>
+        <p class="b4">SEE LOCATIONS</p>
+        <img class="i3" src="Employee.png" alt="Fitzone employee">
+      </div>
+
+      <div class="footer">
+        <p class="fl1">MEMBERSHIPS<br>Cancelation<br>Plans</p>
+        <p class="fl2">RESOURCES<br>Locations<br>Programs</p>
+        <p class="fl3">CONTACT US<br>613-123-4567<br>FitZone@gmail.com</p>
```

Comment:
There is markup that you can use to markup a telephone number and email address.